app/experimenter/nimbus-ui/src/components/PageRequestReview/index.tsx

```diff
@@ -161,6 +162,46 @@ const PageRequestReview = ({
               </ChangeApprovalOperations>
             )}
 
+            <h5>Recommended actions before launch</h5>
+            <Table bordered data-testid="table-signoff" className="mb-4">
```

Comment:
This is not entirely necessary but it might be nice to have this as a separate component, mostly because the Request Review page has a ton of stories already and I'd like to see the various states reflected there!

<img width="1171" alt="image" src="https://user-images.githubusercontent.com/13018240/117186080-1befb900-ada0-11eb-9ce7-9623ceca6e29.png">

Reply:
Yeah I wasn't sure if it was worth pulling into its own component

Reply:
Actually maybe you can sneak it in like, "Preview status, one recommended sign off," since you already have "Review with recommended sign offs" which I somehow missed since the story list there is so long😂